containers/payments/Payment.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { c } from 'ttag';
+import { classnames } from '../../helpers/component';
```

Comment:
And you don't need this line.

Reply:
Done: https://github.com/ProtonMail/react-components/pull/266/commits/b8eb830b7129fc6dfc70a4bad7ecc31d9e4c2bb3